scripts/blender/nerfstudio_blender.py

```diff
@@ -1,43 +1,52 @@
+"""
+nerfstudio_blender.py
+"""
+
 bl_info = {
     "name": "Nerfstudio Add-On",
-    "description": "Create a Nerfstudio JSON camera path from the Blender camera path or import a Nerfstudio camera path as a Blender camera to composite Blender renders over a NeRF background render",
+    "description": "Create a Nerfstudio JSON camera path from the Blender camera path \
+    or import a Nerfstudio camera path as a Blender camera to composite Blender renders \
+    over a NeRF background render for VFX",
     "author": "Cyrus Vachha",
     "version": (1, 0),
-    "blender": (3, 0, 1),
+    "blender": (3, 0, 0),
     "category": "Nerfstudio",
 }
 
-import json
-from math import atan, degrees, radians, tan
+import json  # pylint: disable=wrong-import-position
```

Comment:
I think `# pylint: disable=wrong-import-position` can be added once at the top of the file